google-cloud-clients/google-cloud-translate/src/main/java/com/google/cloud/translate/Translate.java

```diff
@@ -116,15 +116,35 @@ public static TranslateOption format(String format) {
    *
    * <p>Example of listing supported languages, localized according to
    * {@link TranslateOptions#getTargetLanguage()}:
-   * <pre> {@code
-   * List<Language> languages = translate.listSupportedLanguages();
-   * }</pre>
+   * <!--SNIPPET translate_list_codes-->
+   * <pre>{@code
+   *    // TODO(developer): Uncomment these lines.
+   *    // import com.google.cloud.translate.*;
+   *    // Translate translate = TranslateOptions.getDefaultInstance().getService();
+   *
+   *    List<Language> languages = translate.listSupportedLanguages();
+   *
+   *    for (Language language : languages) {
+   *      System.out.printf("Name: %s, Code: %s\n", language.getName(), language.getCode());
```

Comment:
I see the codacy bot is upset by printf below, maybe use `System.out.format`?

Reply:
That warning is thinking that we left in a debug statement. It can be safely ignored for code samples. I think printf will be easier to understand than if we used a logger to hush codacy.